src/common/i18n/locales/en-US.json

```diff
@@ -203,7 +205,10 @@
     "show-history": "Show edit history",
     "hidden-warning": "This post was hidden due to low ratings.",
     "address-copy": "Copy post address",
-    "address-copied": "Post address copied to clipboard."
+    "address-copied": "Post address copied to clipboard.",
+    "browse-original": "Browse to the original post by @{{n}}",
```

Comment:
`Browse original discussion` or `Open original post`, I like first one, what do you think?

Reply:
"Open original post" makes sounds more appropriate .

Reply:
ok, let's choose that and after main category fix, we can merge this PR 👍

Reply:
hold on